angelsbioprocessing/prototypes/overrides/bio-processing-override-bobmodules.lua

```diff
@@ -312,13 +312,36 @@ if mods["bobmodules"] then
       name = "angels-bio-yield-module-3",
       ingredients = {
         {"!!"},
-        {type = "item", name = "solder", amount = 3},
+        --{type = "item", name = "solder", amount = 3},
         {type="item", name="productivity-module-3", amount = 1},
         {type="item", name="effectivity-module-3", amount = 1},
         {type = "item", name = "token-bio", amount = 1}
       }
     }
   })
+  --adding solder when bobplates is active
+if mods["bobplates"] then
+  OV.patch_recipes({
+    {
+      name = "angels-bio-yield-module",
+      ingredients = {
+        {type = "item", name = "solder", amount = 1}
+      }
+    },
+    {
+      name = "angels-bio-yield-module-2",
+      ingredients = {
+        {type = "item", name = "solder", amount = 2}
+      }
+    },
+    {
+      name = "angels-bio-yield-module-3",
+      ingredients = {
+        {type = "item", name = "solder", amount = 3}
+      }
+    }
+  })
```

Comment:
Instead of depending on bobplates, wouldn't it be better to do it when `angelsmods.trigger.smelting_products["solder"] == true` ?